fusedrug/data/tokenizer/modulartokenizer/configs/train_config.yaml

```diff
@@ -0,0 +1,98 @@
+paths:
```

Comment:
This is not relevant here (in fusedrug/data), right?

Reply:
train_config was supposed to have been for an example of use, and tokenizer_config - only for creation, but they're basically the same, so I'll remove it.